crates/bevy_utils/src/num.rs

```diff
@@ -0,0 +1,85 @@
+macro_rules! impl_non_max_fmt {
```

Comment:
There is a `nonmax` crate that already implements this. We should try using that first.